### PR TITLE
A new theme: hexo-theme-Minimalist

### PR DESCRIPTION
**Pull request by CN-DXTZ**
## Check List

**Please read and check followings before submitting a PR.**

- [x] I want to publish my theme on Hexo official website.
    - [x] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [x] `name` is unique.
    - [x] `link` URL is correct.
    - [x] `preview` URL is correct.
    - [x] `preview` URL web site is rendered correctly.
    - [x] Add a screenshot to `source/themes/screenshots`.
    - [x] Screenshot filename is same as value of `name`.
    - [x] Screenshot size is `800 * 500`.
    - [x] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)
    - Languages:
    - [ ] `en` English
    - [ ] `ko` Korean
    - [ ] `pt-br` Brazilian Portuguese
    - [ ] `ru` Russian
    - [ ] `th` Thai
    - [ ] `zh-cn` simplified Chinese
    - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

**Comment by CN-DXTZ** [comment]
Thanks

File: source/_data/themes.yml [hunk 1]
```diff
@@ -3076,3 +3076,13 @@
     - hexo-theme
     - simple
     - clean
+
+- name: Minimalist
+  description: A minimalist style of pure green Hexo theme
+  link: https://github.com/CN-DXTZ/hexo-theme-Minimalist
+  preview: https://cn-dxtz.github.io
+  tags:
+    - Minimalist/极简
```
Comment (noraj) [inline source/_data/themes.yml]: image

Looks good except for the tags

Comment (CN-DXTZ) [reply source/_data/themes.yml]: Thanks, I have deleted the error of tags

Comment (SukkaW) [reply source/_data/themes.yml]: @CN-DXTZ Please avoid using Chinese in tags and descriptions.

**Pull request closed without merging**